internal/sdkprovider/service/project/organization_project.go

```diff
@@ -0,0 +1,276 @@
+package project
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	avngen "github.com/aiven/go-client-codegen"
+	"github.com/aiven/go-client-codegen/handler/organizationprojects"
+	retryGo "github.com/avast/retry-go"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/samber/lo"
+
+	"github.com/aiven/terraform-provider-aiven/internal/common"
+	"github.com/aiven/terraform-provider-aiven/internal/plugin/util"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil"
+	"github.com/aiven/terraform-provider-aiven/internal/schemautil/userconfig"
+)
+
+var ErrProjectStructureChangeNotSupported = fmt.Errorf("modifying parent_id, or billing_group_id is not supported - please use Aiven Console at https://console.aiven.io/ to perform this operation")
+
+var aivenOrganizationProjectSchema = map[string]*schema.Schema{
+	"organization_id": {
+		Type:        schema.TypeString,
+		Description: "ID of an organization. Changing this property forces recreation of the resource.",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"project_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Unique identifier for the project that also serves as the project name. Once set, cannot be changed.",
+	},
+	"billing_group_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Billing group ID to assign to the project. It's required when moving projects between organizations.",
```

Comment:
nit: do I need it if I'm not going to move it?

Reply:
Removed for now. I'll add it later with the future updates